storage/database/dynamodb.go

```diff
@@ -513,6 +537,7 @@ func (batch *dynamoBatch) Put(key, val []byte) error {
 	}
 
 	// If the size of the item is larger than the limit, it should be handled in different way
+	dynamoBatchWriteSizeMeter.Mark(int64(dataSize))
```

Comment:
`dataSize` denotes the value length of a single item. What about removing the metric or changing it to batch item count?

Reply:
You're right. I will remove this metric from both LevelDB and DynamoDB.